jsapp/js/components/permissions/sharingForm.es6

```diff
@@ -118,16 +119,19 @@ class SharingForm extends React.Component {
 
         {stores.session.currentAccount.extra_details?.require_auth !== true &&
           <bem.FormModal__item>
-            <bem.FormView__cell m='warning'>
-              <i className='k-icon k-icon-alert' />
-              <p>
-                {t('Anyone can see this blank form and add submissions to it because you have not set ')}
-                <a href={`/#${ROUTES.ACCOUNT_SETTINGS}`}>
-                  {t('your account')}
-                </a>
-                {t(' to require authentication.')}
-              </p>
-            </bem.FormView__cell>
+            <InlineMessage
+              type='warning'
+              icon='alert'
+              message={
+                <span>
+                  {t('Anyone can see this blank form and add submissions to it because you have not set ')}
+                  <a href={`/#${ROUTES.ACCOUNT_SETTINGS}`}>
+                    {t('your account')}
+                  </a>
+                  {t(' to require authentication.')}
```

Comment:
not new to this PR, but unacceptable use of translated strings. the translator needs to see the entire string, not fragments of it before and after the hyperlink. in this case, it doesn't seem like a big deal to add the `<a href=…` using string replacement instead of JSX, but we may need a more standard way to handle this situation where a `## placeholder ##` in a translated string needs to be replaced by rendered JSX. first thought: split the translated string (*after* translation) into the parts before and after `## placeholder ##`

Reply:
https://chat.kobotoolbox.org/#narrow/stream/4-Kobo-Dev/topic/translated.20sentences.20with.20html.20links

Reply:
I think translators can work around an HTML link i.e.

```
t('Anyone can see this blank form and add submissions to it because you have not set
<a href="##accountsettingsurl##">your account</a> to require authentication.')}
```

But this will require using React's `dangerouslySetInnerHTML` syntax. (Which I think is already needed inside `InlineMessage` )